lib/publishToS3.ts

```diff
@@ -94,14 +94,19 @@ export function executePublishToS3(inputParams: PublishToS3Options): ExecuteGoal
             if (!inv.id.sha) {
                 return { code: 99, message: "SHA is not defined. I need that" };
             }
+            if (inputParams.proxy) {
+                AWS.config.update({
+                    httpOptions: { agent: new proxy(inputParams.proxy) as unknown as httpsAgent | httpAgent },
```

Comment:
This line feels very strange to me since the output of `new proxy` could be httpsAgent or httpsAgent.  I'm going to just remove this and do `as any`.